functions/sql-update/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const process = require('process')
+const sqlService = require('../lib/sql/sql.service')
+const { DONT_DROP_REDIS, TYPES } = sqlService
+
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config()
+}
+
+const schema = `[mtc_admin]`
+
+module.exports = async function (context, sqlUpdateMessage) {

```

Comment:
please can you follow the existing convention for functions, which encapsulates the function definition into a `v1.js` file with a single `process` method.  See some of the existing functions for an example

Reply:
I've done this - however, when I added mine, I happened to look at `check-started` which doesn’t use a separate `v1.js` file. Neither do `diag` or `feedback`. Looking at the latest master for any updates, they all still just use `index.js` directly (as I had done) without splitting out into a `v1.js` file 🤔 

Am I looking in the right place?

Reply:
prepare-check is a good example that does use the preferred arrangement.

those 3 need bringing up to date, i will create a work item for this.